settings/common.py

```diff
@@ -217,3 +220,17 @@
 TWITTER_CONSUMER_SECRET = os.environ.get("TWITTER_CONSUMER_SECRET", "")
 TWITTER_ACCESS_TOKEN = os.environ.get("TWITTER_ACCESS_TOKEN", "")
 TWITTER_ACCESS_TOKEN_SECRET = os.environ.get("TWITTER_ACCESS_TOKEN_SECRET", "")
+
+# Celery
+djcelery.setup_loader()
+
+BROKER_URL = 'redis://localhost:6379/0'
```

Comment:
@vigneshsarma @kracekumar Can you guys confirm the setting ? how do you guys want to handle it